src/__integ__/customEvents.test.ts

```diff
@@ -127,6 +127,13 @@ test('when a plugin calls recordEvent then the event is recorded', async (t: Tes
 });
 
 test('when a plugin calls recordEvent x times then event is recorded x times', async (t: TestController) => {
+    // This test passes individually, but fails on Edge when run as apart of the suite
```

Comment:
There is an underlying problem in this suite of integration tests for custom events, that was somehow exacerbated by this PR. Individually, these tests are fine, but have an order-dependent failure when as a suite on Microsoft Edge. Further investigation is needed to root cause this. But for now, I have disabled just these two tests on Edge to mitigate the immediate problem.

Reply:
This might be related to the order that the tests are run. You can confirm this by removing tests from the suite until this test passes; try to find a single test that, when run before this one, causes the failure.

A common cause is that a test resource isn't being cleaned up properly.

Reply:
Played around with the order like you suggested, but it doesn't look like a normal out-of-order issue. Seems like the issues prop up no matter what order. Also tried separating the tests into two separate files, but that didn't fix anything. Very frustrating :/